packages/apps/webpack.config.js

```diff
@@ -23,7 +23,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
 
   return {
     context,
-    devtool: isProd ? 'source-map' : false,
+    devtool: 'source-map',
```

Comment:
Use `cheap-eval-source-map` for dev-mode (faster compiles), `source-map` for prod.

Alternatively (have not looked into pros/cons it is probably better to just use what react-create-app uses, https://github.com/facebook/create-react-app/blob/next/packages/react-scripts/config/webpack.config.dev.js#L80

(That has been battle-tested)